### ISSUE TITLE
Simbrief generation

### ISSUE THREAD

**Issue by BurtyREDS** [albar965/littlenavmap#944]
Hello Alex, and thanks again for the hard work.
Is it possible to include a quick Simbrief flight plan generation, just entering the type of plane, the departure and arrival airports ?
In fact, Volanta propose this functionnality and I think it is the main reason of its success.
The flow should be : Type of plane + departure + arrival => simbrief background generation => download generated plan to LittleNavMap
Have a nice day
Alberto

**Comment by albar965** [comment]
Have to check how to do this with the SimBrief API.

Not so high priority since it's not that LNM cannot generate flight plans ...

**Comment by BurtyREDS** [comment]
True, but many appreciate generating a Simbrief plan with just a few clicks and importing that plan back into their flight tracking software. It was just a "marketing" suggestion.
